go-controller/cmd/ovnkube-trace/ovnkube-trace.go

```diff
@@ -1179,49 +1179,13 @@ func main() {
 	udp := flag.Bool("udp", false, "use udp transport protocol")
 	addressFamily := flag.String("addr-family", ip4, "Address family (ip4 or ip6) to be used for tracing")
 	skipOvnDetrace := flag.Bool("skip-detrace", false, "skip ovn-detrace command")
+	dumpVRFTableIDs := flag.Bool("dump-udn-vrf-table-ids", false, "Dump the VRF table ID per node for all the user defined networks")
```

Comment:
add a new docs section similar to https://ovn-kubernetes.io/troubleshooting/ovnkube-trace but a totally new udn.md file under troubleshooting and start with this command flag explanation along with a sample output

Reply:
Done